includes/net-standard-1.1.md

```diff
@@ -0,0 +1,18 @@
+.NET Standard 1.1 has 10,239 of the 37,118 available APIs.
+
+| .NET implementation         | Version support                                              |
+|-----------------------------|--------------------------------------------------------------|
+| .NET and .NET Core          | 1.0, 1.1, 2.0, 2.1, 2.2, 3.0, 3.1, 5.0, 6.0                  |
+| .NET Framework <sup>1</sup> | 4.5, 4.5.1, 4.5.2, 4.6, 4.6.1, 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8 |
+| Mono                        | 4.6, 5.4, 6.4                                                |
+| Xamarin.iOS                 | 10.0, 10.14, 112.16                                          |
+| Xamarin.Mac                 | 3.0, 3.8, 5.16                                                          |
+| Xamarin.Android             | 7.0, 8.0, 10.0                                               |
+| Universal Windows Platform  | 8.0, 8.1, 10.0, 10.0.16299, TBD                              |
+| Unity                       | 2018.1                                                       |
+
+<sup>1</sup> The versions listed for .NET Framework apply to .NET Core 2.0 SDK and later versions of the tooling. Older versions used a different mapping for .NET Standard 1.5 and higher. You can [download tooling for .NET Core tools for Visual Studio 2015](https://github.com/dotnet/core/blob/main/release-notes/download-archives) if you cannot upgrade to Visual Studio 2017 or a later version.
```

Comment:
Same here and for other < 1.5 tables.

Reply:
I believe this is applicable, but I'm curious - why wouldn't it be? I might be missing something.

Reply:
Because the different mapping only applies to .NET Standard 1.5 and higher.

Reply:
I'll remove this for refs below 1.5, thanks